orange/src/main/java/com/hartwig/hmftools/orange/algo/sigs/SigsEtiologiesLoader.java

```diff
@@ -0,0 +1,40 @@
+package com.hartwig.hmftools.orange.algo.sigs;
+
+import static com.hartwig.hmftools.common.utils.file.FileDelimiters.TSV_DELIM;
+import static com.hartwig.hmftools.common.utils.file.FileReaderUtils.createFieldsIndexMap;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+import java.util.Map;
+
+import com.google.common.collect.Maps;
+
+import org.jetbrains.annotations.NotNull;
+
+public class SigsEtiologiesLoader
```

Comment:
Class should be final (not meant to be extended, its the equivalent of "object" in kotlin)

Reply:
I've also added a `SigsEtiologiesLoaderTest` btw, since I thought it was missing